rosdep/base.yaml

```
@@ -7333,6 +7333,15 @@ sysstat:
   nixos: [sysstat]
   openembedded: [sysstat@openembedded-core]
   ubuntu: [sysstat]
+systemd:
+  arch: [systemd]
+  debian: [systemd]
+  fedora: [systemd]
+  gentoo: [sys-apps/systemd]
+  macports: [systemd]
```

macports is not the same than homebrew

```suggestion
  osx:
    homebrew:
      packages: [systemd]
```